lib/asls/server.ex

```diff
@@ -73,55 +72,29 @@ defmodule AssemblyScriptLS.Server do
         }
     end
 
-    {:reply, payload, state}
+    # FIXME: Ensure that runtime dependencies are met
+    # before blindly initializing the server
+    {:ok, rt} = Runtime.ensure(params[:rootUri])
+    {:reply, payload, %{state | runtime: rt}}
   end
 
   @impl true
   def handle_call({:notification, %Notification{method: "initialized"}}, _from, state) do
     {:reply, :ok, %{state | initialized: true}}
   end
 
-  @impl true
-  def handle_call({:notification, %Notification{method: "workspace/didChangeConfiguration"}}, _from, state) do
```

Comment:
Worth checking if on this even we have access to any open documents, if that's the case we'll need to fire an analysis for each doc